.github/workflows/barista-preview.yml

```diff
@@ -8,33 +8,23 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-latest
+    container: designops/workspace-base:latest
     env:
-      CI: true
```

Comment:
CI true is now set in the workspace-base, rigth? That's why we can remove it here.

Reply:
We don't need it